app/src/main/java/org/simple/clinic/sync/indicator/SyncIndicatorViewController.kt

```diff
@@ -114,6 +120,7 @@ class SyncIndicatorViewController @Inject constructor(
       dataSync
           .streamSyncErrors()
           .take(1)
+          .filter { error -> error::class in errorTypesToShowErrorFor }
```

Comment:
How are other errors handled? Ignored?

Reply:
Other errors are passed down the UI layer, where it's handled in a `when` block. Since `ResolvedError` is a sealed class, new error types being added will result in a compilation error.

We only filter this here because the `Unauthorized` error is being handled at a higher level (the application level) instead of in this screen.